packages/kit/test/apps/basics/test/cross-platform/test.js

```diff
@@ -962,6 +962,18 @@ test.describe('Routing', () => {
 			expect(await page.textContent('h1')).toBe('symlinked');
 		});
 	}
+
+	test('trailing slash is correct in url.pathname in __data.json (always)', async ({ request }) => {
+		const r = await request.get('/routing/trailing-slash-ssr/always/__data.json');
+		const data = await r.json();
+		expect(data.nodes[1].data[1]).toBe('/routing/trailing-slash-ssr/always/');
```

Comment:
This relies on the internal structure of the `__data.json` requests. Can we instead do a more high-level test which is:
1. start on `trailing-slash-ssr` (btw I would rename this to `trailing-slash-server`)
2. use link clicks to go to different page with trailingSlash config
3. test that the element that prints `$page.url.pathname` / returned `data.pathname` have the desired value

Basically similar to the current `routing/trailing-slash` tests